src/methods/auth.rs

```diff
@@ -91,17 +91,34 @@ impl AuthenticationMethod {
 
     fn parse_continuation(&self, continuation: &str, config: &CoreConfig) -> String {
         if continuation.starts_with("tel:") && self.shim_tel_url {
+            let token = sign_continuation(continuation, config);
             format!(
-                "{}/shim/tel.html?{}",
-                config.server_url(),
-                urlencoding::encode(continuation)
+                "{}{}",
+                config.ui_tel_url(),
+                urlencoding::encode(&token)
```

Comment:
jwts are already URL safe
```suggestion
                &token
```